src/components/Navbar/NavbarMobile.tsx

```diff
@@ -0,0 +1,137 @@
+"use client";
+
+import Image from "next/image";
+import { useEffect,  useState } from "react";
+import { IoClose, IoMenu } from "react-icons/io5";
+import { Outfit } from "next/font/google";
+import Link from "next/link";
+import Login from "../GoogleAuth";
+const outfit = Outfit({
```

Comment:
Not required. See how Outfit font is used in other pages like User Registration Page.